js/modules/networkgraph/networkgraph.src.js

```diff
@@ -437,12 +437,13 @@ seriesType('networkgraph', 'line', {
     },
     // Links:
     getLinkAttribues: function () {
-        var linkOptions = this.series.options.link;
+        var linkOptions = this.series.options.link,
+            pointOptions = this.options;
 
         return {
-            'stroke-width': linkOptions.width,
-            stroke: linkOptions.color,
-            dashstyle: linkOptions.dashStyle
+            'stroke-width': pick(pointOptions.width, linkOptions.width),
+            stroke: pointOptions.color || linkOptions.color,
+            dashstyle: pointOptions.dashStyle || linkOptions.dashStyle
```

Comment:
What is the reason of using the `pick()` for `stroke-width` but not for others?

Reply:
Link's width can be set to 0, but color and dashStyle can not.